build.gradle

```diff
@@ -88,20 +88,6 @@ task release(type: Copy, group: 'build') {
     includeEmptyDirs = false
     // ES versions < 6.3 have a top-level elasticsearch directory inside the plugin zip which we need to remove
```

Comment:
Line unnecessary as we do not support < 6.3

Reply:
Done